exercises/practice/grade-school/GradeSchoolTest.php

```diff
@@ -41,26 +19,96 @@ protected function setUp(): void
         $this->school = new School();
     }
 
-    public function testAddStudent(): void
+    /**
```

Comment:
The first test from canonical data is missing:

```suggestion
    /**
     * uuid: a3f0fb58-f240-4723-8ddc-e644666b85cc
     * @testdox Roster is empty when no student is added
     */
    public function testRosterIsEmptyWhenNoStudentIsAdded(): void
    {
        $school = new School();

        $this->assertSame([], $school->roster());
    }

    /**
```